### PR TITLE
Epic 7 - Yurii Poliuha

### PR DESCRIPTION
**Pull request by yur1k14**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/yurii_poliuha/Epic 7/practise_yurii_poliuha.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <cmath>
+#include <iomanip>
+using namespace std;
+
+void Func();
+void Func2(double array[][10], int Size);
+int* calculateSums(double array[][10], int Size);
+
+int main() {
```
Comment (fm2lpnu) [inline ai_12/yurii_poliuha/Epic 7/practise_yurii_poliuha.cpp:10]: Few subtasks (4,5,6) are not implemented.

File: ai_12/yurii_poliuha/Epic 7/practise_yurii_poliuha.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    const int n = 1016;
+    if (n % 2 == 0)
+        Func();
+
+    return 0;
+}
+
+void Func() {
+    double a = 0.5, b = 3.1, x = 1.4;
```
Comment (fm2lpnu) [inline ai_12/yurii_poliuha/Epic 7/practise_yurii_poliuha.cpp:19]: Comments about used criterias has not been added

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**